### PR TITLE
i18n: Add Japanese translations for R2S, R3S, R4S triggers

### PR DESCRIPTION
**Pull request by niaka3dayo**
This pull request adds missing Japanese translations for R2S, R3S, R4S triggers.

### CODE REVIEW

File: ui/raidboss/data/07-dt/raid/r3s.ts [hunk 1]
```diff
@@ -87,11 +89,13 @@ const triggerSet: TriggerSet<Data> = {
       name: {
         en: 'Barbarous Barrage Uptime Knockback',
         de: 'Brutalo-Bomben Uptime Rückstoß',
+        ja: 'ボンバリアンボム吹き飛ばし無効のタイミング',
```
Comment (MaikoTan) [inline ui/raidboss/data/07-dt/raid/r3s.ts:92]: This function would notify the first tower that would actually knock you back (i.e. which time your アームズレングス or 堅実魔 loses) when it is active.
So I think it should be `ボンバリアンボム吹き飛ばし無効スキル時間切れタイミングを調整`

File: ui/raidboss/data/07-dt/raid/r3s.ts [hunk 2]
```diff
@@ -87,11 +89,13 @@ const triggerSet: TriggerSet<Data> = {
...
         cn: '击退塔uptime打法击退提示时机调整功能',
       },
       comment: {
         en: 'Select towers to dodge with knockback immunity.',
         de: 'Wähle welche Türme mit Rückstoß-Immunität genommen werden.',
+        ja: 'ノックバック無効で避ける塔を選択してください',
```
Comment (MaikoTan) [inline ui/raidboss/data/07-dt/raid/r3s.ts:98]:
```suggestion
        ja: '吹き飛ばし無効で避ける塔を選択してください',
```

Dispite you are using 吹き飛ばし above, should be better to keep it consistence.

**Review by MaikoTan — CHANGES_REQUESTED** [review]
Minor comment

File: ui/raidboss/data/07-dt/raid/r3s.ts (hunk 1 above)
Comment (niaka3dayo) [reply ui/raidboss/data/07-dt/raid/r3s.ts]: > This function would notify the first tower that would actually knock you back (i.e. which time your アームズレングス or 堅実魔 loses) when it is active. So I think it should be something like `ボンバリアンボム吹き飛ばし無効スキル時間切れタイミングを調整` (sounds extremely long, but I have no a better shorter idea)

Thank you for your comment and suggestion. This is indeed a challenging translation. After careful consideration, I've created a commit with the translation 'ボンバリアンボムのアムレン/堅実 限界タイミング通知'. I believe this version effectively conveys the intended meaning.

commit: 43c7ff93b54cc4bd643cba058ed7cf5eafe0ad66

File: ui/raidboss/data/07-dt/raid/r3s.ts (hunk 2 above)
Comment (niaka3dayo) [reply ui/raidboss/data/07-dt/raid/r3s.ts]: > Dispite you are using 吹き飛ばし above, should be better to keep it consistent.

Agreed, thank you for catching this consistency issue. I'll implement your suggested change as is.

commit: 52e01788579e959a054d392ceb42c1c8d4b92482

File: ui/raidboss/data/07-dt/raid/r3s.ts (hunk 1 above)
Comment (MaikoTan) [reply ui/raidboss/data/07-dt/raid/r3s.ts]: This is much better, thanks!

**Review by MaikoTan — APPROVED** [review]

**Pull request merged**